pulp_deb/app/serializers/content_serializers.py

```diff
@@ -273,6 +277,38 @@ def from822(cls, data, **kwargs):
                 # also save the fields not in TRANSLATION_DICT
                 args["custom_fields"][k] = v
 
+        # Delete package fields with values of incorrect type
+        if "installed_size" in args and not args["installed_size"].strip().isdigit():
+            log.warn(
+                ("Removing 'Installed-Size' field from {} as it contains incorrect type.").format(
+                    args["package"]
+                )
+            )
+            del args["installed_size"]
+        if "essential" in args and not isinstance(args["essential"], bool):
+            log.warn(
+                ("Removing 'Essential' field from {} as it contains incorrect type.").format(
+                    args["package"]
+                )
+            )
+            del args["essential"]
+        if "build_essential" in args and not isinstance(args["build_essential"], bool):
+            log.warn(
+                ("Removing 'Build Essential' field from {} as it contains incorrect type.").format(
+                    args["package"]
+                )
+            )
+            del args["build_essential"]
+        if "multi_arch" in args and args["multi_arch"] not in [
+            value[0] for value in BasePackage.MULTIARCH_CHOICES
```

Comment:
```suggestion
            value[1] for value in BasePackage.MULTIARCH_CHOICES
```
Comparing with the `BOOL_CHOICES` this is probably more correct even though it does not make a difference for `MULTIARCH_CHOICES`...